### PR TITLE
Develop mstojanovic add extern ref approved diff list

### PR DESCRIPTION
**Pull request by looser990**

### CODE REVIEW

File: java/messages/src/main/java/com/nedap/retail/messages/epc/v2/approved_difference_list/ApprovedDifferenceListSummary.java [hunk 1]
```diff
@@ -30,6 +30,7 @@
     public static final String MINUS_DIFFERENCE = "minus_difference";
     public static final String EXPORT_STATUS = "export_status";
     public static final String EXPORT_TIME = "export_time";
+    public static final String EXTERM_REF = "extern_ref";
```
Comment (stanislavgrujic) [inline java/messages/src/main/java/com/nedap/retail/messages/epc/v2/approved_difference_list/ApprovedDifferenceListSummary.java]: Typo -> EXTERM_REF

**Comment by nbozic** [comment]
Please fix merge conflicts, other than that looks fine to me

File: java/messages/pom.xml [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.nedap.retail.services</groupId>
     <artifactId>messages</artifactId>
-    <version>1.25</version>
+    <version>1.26</version>
```
Comment (jasperkremer) [inline java/messages/pom.xml]: For retail-services-API I want to use the Artifact plugin for Jenkins, so you don't need to update version number here.

**Comment by jasperkremer** [comment]
Merged as squashed commit.

**Pull request closed without merging**